ontology/yaml/resources/HVAC/entity_types/CHWS.yaml

```diff
@@ -436,3 +436,11 @@ CHWS_SWTC_WDPC_BYPVPM_WFRM_NONCANONICAL_1:
   uses:
   - production_differential_pressure_sensor
   - supply_water_flowrate_sensor # Need to rename to flowrate_sensor
+
+CHWS_RWTM:
```

Comment:
does the CHWS have any temperature control or only monitoring?

Reply:
only monitoring

Reply:
It there a setpoint in the bms that is not bacnet visible? I do not understand how the water system can be not controling to anything.